services/importer/lib/importer/connectors/cdb_data_library_connector.rb

```diff
@@ -104,7 +104,7 @@ def run_create_foreign_table
           execute_as_superuser create_foreign_table_command
         end
         execute_as_superuser %{
-          CREATE VIEW "#{@user.database_schema}".#{foreign_table_name}
+          CREATE OR REPLACE VIEW "#{@user.database_schema}".#{foreign_table_name}
```

Comment:
I am very concerned about - CREATE OR REPLACE VIEW change..
If we can rather swallow the exception without this change, it is better..
It has some association with changing columns of a published dataset..
Already mentioned to Andrew.. Just logging the same..